packages/esm-ohri-pmtct-app/src/pmtct/home-dashboard/mother-child-summary-tabs.component.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import { Tabs, Tab, TabPanels, TabPanel, TabList } from '@carbon/react';
+import styles from './ohri-patient-tabs.scss';
```

Comment:
Let's move this style import at the bottom of the imports

Reply:
Done